specs/protocol/span-batches.md

```diff
@@ -30,13 +30,9 @@
 
 [g-deposit-tx-type]: ../glossary.md#deposited-transaction-type
 
-> The span-batches spec is experimental :shipit:
->
-> \*this feature is in active R&D and not yet part of any hard fork
-
 ## Introduction
 
-Span-batches reduce overhead of OP-stack chains.
+Span-batch is a new batching spec that reduces overhead of OP-stack chains, introduced in Delta hardfork.
```

Comment:
Could you cross link to the delta upgrade section & change `hardfork` to `network upgrade`